compiler/src/main/resources/compiling-programs/multi-state-index.cell

```diff
@@ -0,0 +1,14 @@
+world {
+    size = 100 [wrap], 200 [wrap]
+    tickrate = 1
+    cellsize = 5
+}
+
+state ident (0, 0, 0) {
+    // Assumes that the semantics of a hashtag is also the id of a given state in a non-multi-state state, which would be 0
+    if (# == 2) {
```

Comment:
So what about multi-state when it *is* a multi-state declaration?

Reply:
Yes, good point. Fixed in b3585ac

Reply:
`state ident 5 (0, 0, 0) { ... }` shouldn't compile?? The syntax is `state ident[5] (0, 0, 0) { ... }`

Reply:
Right. Fixed in 7e14c7f